src/main/resources/locale_de_DE.properties

```
@@ -198,5 +189,19 @@ words.refMessage=Referenz Nachricht
 words.reputation=Reputation
 words.role=Rolle
 words.unknown=Unbekannt
+command.reaction.checking=Überprüfe Reaction
+command.reaction.description=Verwalte die Reputation Reactions
+command.reaction.sub.add=Füge eine Reaction für Reputation hinzu
+command.reaction.sub.add.add=Reaction %EMOTE% wurde hinzugefügt
+command.reaction.sub.info=Informationen über aktuell genutzte Reactions.
+command.reaction.sub.info.additional=Zusätzliche Reactions
+command.reaction.sub.info.main=Standard Reaction
+command.reaction.sub.info.title=Registrierte Reactions
+command.reaction.sub.main=Setze die standard Repurationsreaction
+command.reaction.sub.reaction.get.error=Die Reaction existiert auf diesem Server nicht.
+command.reaction.sub.remove=Entferne eine Reaction als Reputationsreaction.
+command.reaction.sub.remove.notFound=Reaction ist nicht registriert.
+command.reaction.sub.remove.removed=Reaction wurde entfernt.
+command.reaction.sub.main.set=Reputations Reaction wurde zu %EMOTE% geändert.
```

In the existing translation, `reaction` was translated with `Reaktion`
```suggestion
command.reaction.checking=Überprüfe Reaktion.
command.reaction.description=Verwalte die Reputations-Reaktion
command.reaction.sub.add=Füge eine Reaktion für Reputation hinzu
command.reaction.sub.add.add=Reaktion %EMOTE% wurde hinzugefügt
command.reaction.sub.info=Informationen über aktuell genutzte Reaktionen.
command.reaction.sub.info.additional=Zusätzliche Reaktionen
command.reaction.sub.info.main=Standard Reaktion
command.reaction.sub.info.title=Registrierte Reaktionen
command.reaction.sub.main=Setze die Standard-Repurationsreaktion
command.reaction.sub.reaction.get.error=Die Reaktion existiert auf diesem Server nicht.
command.reaction.sub.remove=Entferne eine Reaktion als Reputationsreaktion.
command.reaction.sub.remove.notFound=Reaktion ist nicht registriert.
command.reaction.sub.remove.removed=Reaktion wurde entfernt.
command.reaction.sub.main.set=Reputations-Reaktion wurde zu %EMOTE% geändert.
```